src/app/core/store/trips/trips.effects.ts

```diff
@@ -0,0 +1,348 @@
+import { Injectable } from '@angular/core';
+import { catchError, exhaustMap, map, of, tap } from 'rxjs';
+import { Actions, createEffect, ofType } from '@ngrx/effects';
+import { MatSnackBar } from '@angular/material/snack-bar';
+import { TripsService } from '../../../features/trips/services/trips.service';
+import {
+  carriagesCreatedSuccess,
+  carriagesLoadedSuccess, carriageUpdatedSuccess,
+  createCarriage, createOrder, createOrderSuccess, createRide,
+  createRideSuccess, createRoute, createStation, createStationSuccess,
+  deleteOrder, deleteRoute, deleteStation, failureSnackBar, loadCarriages, loadOrders,
+  loadRideById, loadRouteById, loadRoutes, loadSearch, loadStations, loadUsers, orderDeletedSuccess,
+  ordersLoadedSuccess, rideLoadedByIdSuccess, routeCreatedSuccess, routeDeletedSuccess,
+  routeLoadedByIdSuccess, routesLoadedSuccess, routeUpdatedSuccess, searchLoadedSuccess,
+  stationDeleteSuccess, stationsLoadedSuccess, updateCarriage, updateRide, updateRideSuccess,
+  updateRoute, usersLoadedSuccess
+} from './trips.actions';
```

Comment:
I think, for short, you can do the import like this:
```
import * as tripActions from './trips.actions';
```
and use it in code so for exsemple:
```
tripActions.stationsLoadedSuccess({ stations })
```